salalib/ntfp.cpp

```diff
@@ -100,16 +103,16 @@ void NtfMap::open(const std::vector<std::string>& fileset, Communicator *comm)
    time_t time = 0;
    qtimer( time, 0 );
 
-   pvecint featcodes;
+   std::vector<int> featcodes;
 /*
    m_bottom_left.a =  2147483647;   // 2^31 - 1
    m_bottom_left.b =  2147483647;
    m_top_right.a   = -2147483647;
    m_top_right.b   = -2147483647;
 */
    m_line_count = 0;
-   if (size()) {
-      clear();
+   if (!layers.empty()) {

```

Comment:
The empty check is pointless, clear on an empty vector is no more expensive than the isEmpty check.